src/const/api.ts

```diff
@@ -1,7 +1,21 @@
-export const mainLink: string = "https://deckofcardsapi.com/api/deck/";
-export const newDeckShuffledLink: string = "https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=";
-export const drawOneCardLink: string = "/draw/?count=1";
-export const drawTwoCardsLink: string = "/draw/?count=2";
-export const drawXCards: string = "/draw/?count=";
-export const reshuffleDeckLink: string = "/shuffle/"
-export const decksCount: number = 6;
\ No newline at end of file
+import { TEndpoints } from "../types/global";
+
+const mainLink: string = "https://deckofcardsapi.com/api/deck/";
```

Comment:
tu też niepotrzebnie jawnie dajesz typ skoro masz wartość w zmiennej. To jest ok jak się uczymy, ale w projekcie niepotrzebnie wydłużamy kod